monitoring/README.md

```diff
@@ -2,6 +2,8 @@
 
 The deployment uses the [CoreOS Prometheus Operator](https://coreos.com/operators/prometheus/docs/0.15.0/index.html). 
 
+Alertmanager docs: [Alertmanager](https://prometheus.io/docs/alerting/configuration/).
+
 **The monitoring folder contains the following artifacts:**
 * deploy scripts to:
     * deploy the Prometheus Operator along with Grafana and Alert Manager and other Helm charts that help monitor GKE.
```

Comment:
s/Alert Manager/Alertmanager

Reply:
Just an introductory note on Alertmanager would be great: The Alertmanager component of Prometheus handles alerts sent by client applications such as the Prometheus server. It takes care of deduplicating, grouping, and routing them to the correct receiver integration such as email, PagerDuty, or OpsGenie. It also takes care of silencing and inhibition of alerts.

Reply:
Or, just add a second link to the Alerting concepts page: https://prometheus.io/docs/alerting/overview/

Reply:
I just checked and Alertmanager is a single word across all documentation. Not Alert manager